pages/studentInstanceQuestionHomework/studentInstanceQuestionHomework.ejs

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <%- include('../partials/head'); %>
+    <%- include('../partials/head', {pageTitle: instance_question_info.question_number}); %>
```

Comment:
Why is this different to the exam equivalent?

Reply:
That's a big reason why it took so long to make a new PR for this, it's because `instance_question_info.question_number` is formatted completely differently for questions and for exams. Basically, homeworks format that value like "HW1.1" but exams will have it formatted like "1". I tried to fix the inconsistency myself but I couldn't really guarantee that it wouldn't break things elsewhere, so I decided to leave that out of this PR.

Reply:
Sounds good!